code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1000,6 +1001,12 @@
 				H.wash_cream()
 			SEND_SIGNAL(M, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
 
+/datum/reagent/space_cleaner/on_mob_life(mob/living/carbon/M)
+	if(toxpwr)
+		M.adjustToxLoss(toxpwr*REM, 0)
+		. = TRUE
+	..()
+
```

Comment:
```suggestion
	M.adjustToxLoss(toxpwr*REM, 0)
	..()

```
- This if statement is useless because it can never be not true.
- Why are you setting the function to return true? It isn't really expecting a result.